pyzeebe/channel/camunda_cloud_channel.py

```diff
@@ -1,23 +1,31 @@
-from typing import Any, Dict, Optional
+from typing import Optional
 
+import deprecation  # type: ignore[import-untyped]
 import grpc
 from oauthlib import oauth2
 from requests import HTTPError
 from requests_oauthlib import OAuth2Session
 
 from pyzeebe.channel.channel_options import get_channel_options
+from pyzeebe.credentials.typing import ChannelArgumentType
 from pyzeebe.errors import (
     InvalidCamundaCloudCredentialsError,
     InvalidOAuthCredentialsError,
 )
 
 
+@deprecation.deprecated(
```

Comment:
https://typing-extensions.readthedocs.io/en/latest/#typing_extensions.deprecated

Reply:
✅ Used [typing_extensions.deprecated](https://typing-extensions.readthedocs.io/en/latest/#typing_extensions.deprecated) and removed the deprecation package.